packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx

```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
+                <StepCardFade />
+              </StepCard>
+            </StepCardsWrapper>
+          </CenteredLayoutGrid>
+        </StyledLimitedContainerWidth>
+      </ThreeStepsBackgroundContainer>
+      {/* ypp benefits */}
+      <BackgroundContainer noBackground>
+        <StyledLimitedContainerWidth as="section">
+          <CardsWithImagesContainer>
+            <CenteredLayoutGrid>
+              <HeaderGridItem
+                marginBottom={mdMatch ? 24 : 16}
+                as="header"
+                colStart={{ sm: 3, lg: 4 }}
+                colSpan={{ base: 12, sm: 8, lg: 6 }}
+              >
+                <Text variant={mdMatch ? 'h800' : 'h600'} as="h2">
+                  There is a lot more to YouTube Partner Program
+                </Text>
+                <Text variant="t300" as="p" margin={{ top: 4 }} color="colorText">
+                  New to Atlas? Joining our YouTube Partner Program is an exciting opportunity to try out the future of
+                  online video sharing. And we're only getting started.
+                </Text>
+              </HeaderGridItem>
+            </CenteredLayoutGrid>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 1, lg: 2 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer>
+                  <CardImage dropShadow absolute src={dashboardImgBack} alt="Dashboard" />
+                  <CardImage dropShadow src={dashboardImgFront} ref={dashboardImageRef} alt="Reward card" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  title="Manage your channel on a simple dashboard"
+                  subtitle="YPP dashboard"
+                  body="There you can check your channel status and get instructions on completing all actions to get rewards"
+                />
+              </GridItem>
+            </CardImageRow>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 6, lg: 6 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer positionOnMobile="flex-end">
+                  <CardImage absolute dropShadow src={myVideosBack} alt="My videos" />
+                  <CardImage dropShadow src={myVideosFront} ref={myVideosImageRef} alt="Button" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem
+                colStart={{ sm: 3, md: 1, lg: 2 }}
+                rowStart={{ md: 1 }}
+                colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}
+              >
+                <ContentCard
+                  title="Backup your videos and own them forever"
+                  subtitle="YouTube backup"
+                  body="You can sleep peacefully knowing that your conent is safe and forever stored on blockchain. You can even sell it as NFTs — you own it forever."
```

Comment:
```suggestion
                  body="You can sleep peacefully knowing that your content is safe and forever stored on blockchain. You can even sell it as NFTs — you own it forever."
```

Reply:
I used a different copy after all - based on [this frame](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1606%3A85515)